mmv1/products/gkebackup/terraform.yaml

```diff
@@ -0,0 +1,79 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  BackupPlan: !ruby/object:Overrides::Terraform::ResourceOverride
+    timeouts: !ruby/object:Api::Timeouts
+    autogen_async: true
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "gkebackup_backupplan_basic"
+        min_version: beta
+        primary_resource_id: "basic"
+        vars:
+          name: "basicplan"
```

Comment:
Like I mentioned in the handwritten test, we'd likely want `tf-test` prefix for resource name. I believe have `-` in the test variable would automatically generate the prefix in generated test. Like `basicplan` -> `basic-plan` or `basicplan-1` in this case.

Reply:
added to all resource names in the tests.